src/util/lib/cosi/template/fetch/index.js

```diff
@@ -156,7 +156,11 @@ class Fetch extends Events {
             }
 
             // list of all templates applicable to this host
-            const wantTemplates = [ 'check-system', 'worksheet-system' ];
```

Comment:
just dashboard-system the others should be named appropriately after the plugins the metrics come from. if you want a load graph by default add a load plugin in NAD that is installed by default.